### PR TITLE
Added phones with barometric sensor information.

### PR DESCRIPTION
**Pull request by rgmf**
**Describe the pull request**
Added information about smartphones with barometric sensor to README_TESTED_SENSORS.md

**License agreement**
By opening this pull request, you are providing your contribution under the _Apache License 2.0_ (see [LICENSE.md](LICENSE.md)).

**Note: new dependencies/libraries**
Please refrain from introducing new libraries without consulting the team.

### CODE REVIEW

File: README_TESTED_SENSORS.md [hunk 1]
```diff
@@ -56,3 +56,17 @@ Also the distance is not computed.
...
+
+## Tested Barometric Sensor Smartphones
+
+There are smartphones that have a barometric sensor that OpenTracks uses to compute altitude gain and loss.
+
+In our tests there are smartphones whose barometric sensor works well and others don't. Here is a list of the smartphones tested:
+
+|Brand|Model|Android Version|Tests|
+|-----|-----|---------------|-----------|
+|Google|Pixel|LineageOS 17.1|It works well|
```
Comment (dennisguse) [inline README_TESTED_SENSORS.md:68]: s/It works well/ok ?

Comment (dennisguse) [reply README_TESTED_SENSORS.md:68]: s/It malfunctions/failure/

**Review by dennisguse — APPROVED** [review]
I really love the idea :D
I have some trouble on some of the devices I use...

**Comment by rgmf** [comment]
@dennisguse FYI:

I'm not completly sure but smartphones with barometric sensor and waterproof fails computing gain/loss because barometric values change a lot when you pressure them.

If they aren't waterproof then they work very well.

It's only a weak hypothesis.

**Comment by dennisguse** [comment]
Interesting hypothesis!

**Pull request merged**